docs/modules/about/pages/introduction.adoc

```diff
@@ -3,25 +3,25 @@
 
 = What is {page-component-title}?
 
-{page-component-title} is an application that allows users to create flexible dashboards to interact with data stored by OpenNMS. {page-component-title} works with https://grafana.com[Grafana], an analytics platform, to display and customize xref:fault[fault management (_FM_)] and xref:performance[performance management (PM)] data from https://www.opennms.org[OpenNMS(R) Horizon(TM)] and/or https://www.opennms.com[OpenNMS(R) Meridian(TM)]. Create and customize dashboards to display the data you want to see -- alarms, outages, key performance indicators -- in a way that best meets the needs of your workflow and staff. 
+{page-component-title} is an application that lets users to create flexible dashboards to interact with data stored by OpenNMS. {page-component-title} works with https://grafana.com[Grafana], an analytics platform, to display and customize xref:fault[fault management (FM)] and xref:performance[performance management (PM)] data from https://www.opennms.org[OpenNMS(R) Horizon(TM)] and/or https://www.opennms.com[OpenNMS(R) Meridian(TM)]. Create and customize dashboards to display the data you want to see -- alarms, outages, key performance indicators -- in a way that best meets the needs of your workflow and staff. 
```

Comment:
"to create" is the infinitive form of the verb.  "lets users create" would be correct, grammatically.

Reply:
@swachter , as Mark said, "lets users to" is incorrect. Originally it was "allows users to". Our new style guidelines from NH suggest using "lets" instead of "allows". I missed deleting the "to" when I updated it. :-(